packages/bits/src/lib/selector/selector.component.html

```diff
@@ -17,6 +19,7 @@
             [displayStyle]="'default'"
             [icon]="'caret-down'"
             (click)="overlay.toggle()"
+            [ariaLabel]="'Open Selector Menu'"
```

Comment:
Can be changed to `ariaLabel="Open Selector Menu"`

Reply:
make sense